example/index.ts

```diff
@@ -99,7 +99,11 @@ Promise.all([
   }, {
     accountId: 'admin@test',
     pageSize: 5,
-    firstTxHash: undefined
+    firstTxHash: undefined,
```

Comment:
I believe this part needs to be updated to resolve the conflicts between the branches.

```suggestion
    firstTxHash: undefined,
    firstTxTime: undefined,
    lastTxTime: undefined,
    firstTxHeight: undefined,
    lastTxHeight: undefined,
```